src/jit/codegencommon.cpp

```diff
@@ -1732,11 +1732,13 @@ unsigned CodeGenInterface::InferStructOpSizeAlign(GenTreePtr op, unsigned* align
         LclVarDsc* varDsc = compiler->lvaTable + varNum;
         assert(varDsc->lvType == TYP_STRUCT);
         opSize = varDsc->lvSize();
+#ifndef _TARGET_64BIT_
         if (varDsc->lvStructDoubleAlign)
         {
             alignment = TARGET_POINTER_SIZE * 2;
         }
         else
+#endif
```

Comment:
Add trailing comment 
`// _TARGET_64BIT_`

Reply:
Technically, by CC, the comment on endif should match the condition of the preceding section, so
```
// !_TARGET_64BIT_
```
(we generally elide the "defined" in the comments...)

Reply:
Done